spring-integration-file/src/main/java/org/springframework/integration/file/tail/FileTailingMessageProducerSupport.java

```diff
@@ -96,6 +108,11 @@ protected TaskExecutor getTaskExecutor() {
 	}
 
 	@Override
+	public void setTaskScheduler(TaskScheduler taskScheduler) {
```

Comment:
M-m-m. We don't need this setter - `AbstractEndpoint`:
```
@Override
public void setTaskScheduler(TaskScheduler taskScheduler) {
	super.setTaskScheduler(taskScheduler);
}
```
Am I missing anything?